src/sentry/search/events/builder/__init__.py

```diff
@@ -1,5 +1,5 @@
+from .base import BaseQueryBuilder  # NOQA
```

Comment:
please use absolute imports instead of reexports in `__init__.py` -- this makes it really easy to end up with import cycles and impacts startup time / test time since it makes it much more likely that any import pulls in the whole app

Reply:
I'll remove this line for now, and the others in a follow up PR